polaris-react/src/components/Button/Button.scss

```diff
@@ -393,8 +393,6 @@
 
   > .Content {
     font-weight: var(--p-font-weight-regular);
-    // stylelint-disable-next-line -- generated by polaris-migrator DO NOT COPY
-    @include focus-ring;
```

Comment:
I can't think why we'd want button content focusable. It forces the focused state below and results in a double outline.

Reply:
This is for the plain button, it should be left in.
<img width="168" alt="Screenshot 2023-05-31 at 3 09 51 PM" src="https://github.com/Shopify/polaris/assets/3474483/a5a0a1a1-b0b5-41a1-9e96-effa7f0597b3">

Reply:
Can we refactor it at all? We'd prefer the focus on the actual button so we can remove forcing the focus ring where possible and get focus for free via browser / dom stuff